lms/envs/common.py

```diff
@@ -416,6 +416,19 @@
     # .. toggle_status: supported
     # .. toggle_warnings: None
     'ENABLE_CHANGE_USER_PASSWORD_ADMIN': False,
+
+    # .. toggle_name: ENABLE_COURSEWARE_MICROFRONTEND
```

Comment:
I had separately documented the `REDIRECT_TO_COURSEWARE_MICROFRONTEND` toggle in lms/djangoapps/courseware/toggles.py - are the two comments duplicative?  

That is only used in `should_redirect_to_courseware_microfrontend` - I was admittedly pattern matching off of other MFE toggles we had in the platform.

Reply:
We've got separate flags for whether it's enabled at the platform level vs the waffle flag that can be flipped on a per-course basis. Both have similar documentation requirements. I didn't add the flag (it was already in use), but I did make it explicit in the common.py instead of being scanned for with fallback. I'll try to make the comment a little clearer about purpose.